app/models/Item.scala

```diff
@@ -29,6 +29,20 @@ case class Item(id: Int,            // DB key
                 updated: Date,      // Time of last transfer
                 transfers: Int)  {  // Number of transfers
 
+  def collection = {
+    DB.withConnection { implicit c =>
+      SQL("select * from collection where id = {collection_id}")
+      .on('collection_id -> collectionId).as(Collection.coll.single)
+    }
+  }
+
+  def holds = {
```

Comment:
Absolutely right - the problem is that there are 2 conflicting patterns at work: 'holds' was meant to agree with the Item (method/field) 'transfers' which is a count. So I didn't want 'transfers' & 'holdCount' being the same things and sounding different. All those method names need to be made consistent...